packages/management/tkr-conversion-webhook/bundle/config/upstream/tkr-crd

```diff
@@ -0,0 +1,382 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+    cert-manager.io/inject-ca-from: tkr-system/tkr-conversion-webhook-serving-cert
```

Comment:
`tkr-system` is hardcoded here. This needs to be configurable like all the other occurrences of tkr-system.

Reply:
Resolved in 6883ff365e84609aa88ac326af09063a78a271f3